CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 7.3.0
+  - Standardized SSL settings: [#42](https://github.com/logstash-plugins/logstash-mixin-http_client/pull/42)  
+    - Deprecated `cacert` in favor of `ssl_certificate_authorities`
+    - Deprecated`client_cert` in favor of `ssl_certificate`
+    - Deprecated `client_key` in favor of `ssl_key`
+    - Deprecated `keystore` in favor of `ssl_keystore_path`
+    - Deprecated `keystore_password` in favor of `ssl_keystore_password`
+    - Deprecated `keystore_type` in favor of `ssl_keystore_type`
+    - Deprecated `truststore` in favor of `ssl_truststore_path`
+    - Deprecated `truststore_password` in favor of `ssl_truststore_password`
+    - Deprecated `truststore_type` in favor of `ssl_truststore_type`
```

Comment:
Instead of focusing on what we are deprecating, it may be better to focus on what we are _adding_:

```suggestion
  - Adds standardized SSL settings and deprecates their non-standard counterparts. Deprecated settings will continue to work, and will provide pipeline maintainers with guidance toward using their standardized counterparts [#42](https://github.com/logstash-plugins/logstash-mixin-http_client/pull/42)
    - Adds new `ssl_truststore_path`, `ssl_truststore_password`, and `ssl_truststore_type` settings for configuring SSL-trust using a PKCS-12 or JKS trust store, deprecating their `truststore`, `truststore_password`, and `truststore_type` counterparts.
    - Adds new `ssl_certificate_authorities` setting for configuring SSL-trust using a PEM-formated list certificate authorities, deprecating its `cacert` counterpart.
   - Adds new `ssl_keystore_path`, `ssl_keystore_password`, and `ssl_keystore_type` settings for configuring SSL-identity using a PKCS-12 or JKS key store, deprecating their `keystore`, `keystore_password`, and `keystore_type` counterparts.
    - Adds new `ssl_certificate` and `ssl_key` settings for configuring SSL-identity using a PEM-formatted certificate/key pair, deprecating their `client_cert` and `client_key` counterparts. 
```